aptos-move/block-executor/src/executor.rs

```diff
@@ -553,69 +549,48 @@ where
             return Ok(vec![]);
         }
 
-        let num_txns = signature_verified_block.len() as u32;
-        let last_input_output = TxnLastInputOutput::new(num_txns);
-        let scheduler = Scheduler::new(num_txns);
+        let num_txns = signature_verified_block.len();
+
+        let shared_commit_state = Mutex::new((
+            FeeStatement::zero(),
+            Vec::<FeeStatement>::with_capacity(num_txns),
+            None,
+        ));
 
-        let mut roles: Vec<CommitRole> = vec![];
-        let mut senders: Vec<Sender<u32>> = Vec::with_capacity(self.concurrency_level - 1);
-        for _ in 0..(self.concurrency_level - 1) {
-            let (tx, rx) = mpsc::channel();
-            roles.push(CommitRole::Worker(rx));
-            senders.push(tx);
+        let final_results = ExplicitSyncWrapper::new(Vec::with_capacity(num_txns));
+        // Initialize for [] access, takes less than <1ms (on Apple M1) for 50k block.
```

Comment:
let's get rid of this comment and re-check this is fine. The number's coming from me but I quickly timed with a test run, and tests likely use mock types.